package.json

```diff
@@ -85,7 +86,8 @@
     "remove-markdown": "0.1.0",
     "simple-markdown": "^0.4.0",
     "styled-components": "^3.1.4",
-    "styled-system": "^2.2.4"
+    "styled-system": "^2.2.4",
+    "yup": "^0.25.1"
```

Comment:
Any chance you explain your reasoning for adding 2 new dependencies for a single screen?

Reply:
I actually have almost no justifications, but it seems like we use them in other repos already. I also started an [experiment](https://github.com/artsy/emission/commit/23e95bd0df08f0f87a0bd569afed2ec6164de5ac?diff=split) without formik and I have a feeling that it might be better to not depend on at least formik.

Reply:
I'd prefer that, all dependencies are liabilities - we already have an example of a working form [in Metadata.ts](https://github.com/artsy/emission/blob/master/src/lib/Components/Consignments/Screens/Metadata.tsx)